.travis.yml

```diff
@@ -5,6 +5,10 @@ cache:
   directories:
   - node_modules
   - bower_components
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
yeah thought so, but this seemed to make it work (is what i copied from other feeder repos like kve & extension-registry).  Ill dig around a bit more.